client/src/components/Ticketing/ticketingmanager/Events/InputField.tsx

```diff
@@ -9,39 +9,58 @@ interface InputFieldProps
     action(any): void,
     actionType:string,
     placeholder: string,
-    value:string
+    value:string,
+    disabled?:boolean
 }
-const InputFieldForEvent =
-    ({type, name, id, actionType,
-      action, placeholder, headerText, value}: InputFieldProps) => {
-      if (actionType =='onSubmit') {
-        return (
-        // eslint-disable-next-line react/react-in-jsx-scope
-          <>
-            <h3 className='text-sm text-zinc-600 '>{headerText}</h3>
-            <input
-              className='w-full p-2 rounded-lg border border-zinc-300 mb-4'
-              id={id} type='input' name={name}
-              onSubmit={action} placeholder={placeholder}
-              defaultValue={value} value={value}
-              maxLength={255}
-            />
-          </>
-        );
-      } else if (actionType =='onChange') {
-        return (
-          <>
-            <h3 className='text-sm text-zinc-600 '>{headerText}</h3>
-            <input
-              className='w-full p-2 rounded-lg border border-zinc-300 mb-4'
-              id={id} type={type} name={name}
-              onChange={action} placeholder={placeholder}
-              defaultValue={value}
-              maxLength={255}
-            />
-          </>
-        );
-      }
-    };
+const InputFieldForEvent = (props: InputFieldProps) => {
+  const {type, name, id, actionType,
+    action, placeholder, headerText, value, disabled} = props;
+  if (actionType =='onSubmit') {
+    return (
+      // eslint-disable-next-line react/react-in-jsx-scope
+      <>
+        <label
+          htmlFor={id}
+          className='text-sm text-zinc-600 '
+        >
+          {headerText}
+        </label>
+        <input
+          className='w-full p-2 rounded-lg border border-zinc-300 mb-4'
+          id={id}
+          type={type}
```

Comment:
Will the fact that the input type changed from a set `'input'` to being passed in as a prop affect other areas where `InputField` is used?

Reply:
It should be ok, 'input' is not a defined type so the inclusion wasn't doing anything, the tag was just reverting to type text. The prop interface has type as an optional value so if not supplied the tag will just revert to type text again. Also the component is only used in the Event form and actionType is never set to onSubmit, so the entire block could be removed.

Reply:
Ok, sounds good. I think we should still probably keep the `onSubmit` for the time being since it doesn't necessarily hurt.